### ISSUE TITLE
Portage logo does not scale

### ISSUE THREAD

**Issue by jasonbrodeur** [ualbertalib/DMPonline_v4#72]
The portage logo on the home page does not scale with the size of the window - as a result, it overlaps with links and content when a small screen is used.

**Comment by mhvezina** [comment]
Yes, indeed both header and footer responsiveness are not ok... This is a screen capture for a 1024x600 screen:
http://quirktools.com/screenfly/#u=https%3A//portagenetwork.ca/&w=1024&h=600 
You can also test other screen resolutions.

**Comment by mhvezina** [comment]
Additional comment made by D. Sauvé related to this issue:

Tag line under logo in French (“Services partagés pour les données de recherche”) doesn’t fit on one line even on some medium size screens (I have an 17”; see capture).  1) Could the font be slightly condensed and 2), since it will wrap anyway on smaller screen as for the English version, could the spacing between the two lines be smaller so the tag line looks like one statement instead of two paragraphs?
![portage_logo_and_subtitle_not_scaling](https://cloud.githubusercontent.com/assets/10760472/10856780/34e63aec-7f20-11e5-8ae5-501a84703464.png)
